spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,3 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
```

Comment:
I think you need to create a method and initialize the vars used in the real mailer

Reply:
Make sure it works, entering the path above